stable/feeds/tests/secret_test.yaml

```diff
@@ -0,0 +1,26 @@
+suite: Secret Tests
+templates:
+  - secret.yaml
+release:
+  name: test-release
+  namespace: test-namespace
+chart:
+  version: 9.9.9
+  appVersion: 9.9.9
+tests:
+  - it: should render github driver token as empty string when not set
```

Comment:
just to be clear, this doesnt actually set the driver token as an empty string (ie. ANCHORE_GITHUB_TOKEN=""). It doesnt set it at all in the secret, and so it'll use the env_vars value of ANCHORE_GITHUB_TOKEN="default-unset"